.p4a

```diff
@@ -1,8 +1,8 @@
 --window
 --bootstrap "webview"
---package "org.learningequality.Kolibri"
---name "Kolibri"
---dist_name "kolibri"
+--package "org.endlessos.EndlessKey"
+--name "Endless Key"
+--dist_name "EndlessKey"
```

Comment:
I don't know if it's worth making this change. It would require the Makefile targets to change, which broke the Jenkins CI.

Reply:
Just to follow up on this, it might be nice to change `--dist_name` so the completed APK is called `EndlessKey-*.apk`, but let's just punt on that for now.